### PR TITLE
Term edition interface

### PR DESCRIPTION
**Pull request by leonofre**
Related to #97

### CODE REVIEW

File: admin/class-monk-admin.php [hunk 1]
```diff
@@ -679,6 +679,7 @@ public function monk_term_translation_meta_field( $term ) {
 		$taxonomies                = get_taxonomies();
 		$monk_term_translations_id = get_term_meta( $term->term_id, '_monk_term_translations_id', true );
 		$monk_term_translations    = get_option( 'monk_term_translations_' . $monk_term_translations_id, false );
+		$avaliable_languages       = false;
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Small misspelling it's `$available_languages` not ``$avaliable_languages``

File: admin/partials/admin-monk-term-translation.php [hunk 2]
```diff
@@ -19,40 +19,41 @@
...
 
-						if ( $translation_code === $language && $monk_language !== $language ) : ?>
-							<a href="<?php echo esc_url( $translation_term_url ); ?>">
-								<?php echo esc_html( $monk_languages[ $language ]['name'] ); ?>
-							</a>
-						<?php
-						endif;
-					endforeach;
-				endforeach;
+		if ( $translation_code === $language && $monk_language !== $language ) :
```
Comment (brenoalvs) [inline admin/partials/admin-monk-term-translation.php]: Why using 2 `foreach`?
At this line we can check by `in_array( $translation_code, $languages );`

File: admin/partials/admin-monk-term-translation.php [hunk 3]
```diff
@@ -19,40 +19,41 @@
...
+		<?php
+		endif;
+	endforeach;
+endforeach;
+				?>
+				</ul>
+				<?php
+				if ( $avaliable_languages ) :
+					$new_term_url = add_query_arg( array(
+						'monk_term_id'     => $monk_term_translations_id,
```
Comment (brenoalvs) [inline admin/partials/admin-monk-term-translation.php:48]: We're changing this to `monk_id`

File: admin/partials/admin-monk-term-translation.php [hunk 4]
```diff
@@ -19,40 +19,41 @@
...
+					), $base_url );
 				?>
+					<a href="<?php echo esc_url( $new_term_url ); ?>" class="button">Add a translation +</a>
+				<?php else : ?>
+					<p><?php esc_html_e( 'No Avaliable Translations!', 'monk' ); ?></p>
```
Comment (brenoalvs) [inline admin/partials/admin-monk-term-translation.php]: No Available...

File: admin/partials/admin-monk-term-translation.php [hunk 5]
```diff
@@ -19,40 +19,41 @@
...
+endforeach;
+				?>
+				</ul>
+				<?php
+				if ( $avaliable_languages ) :
+					$new_term_url = add_query_arg( array(
+						'monk_term_id'     => $monk_term_translations_id,
+					), $base_url );
 				?>
+					<a href="<?php echo esc_url( $new_term_url ); ?>" class="button">Add a translation +</a>
```
Comment (brenoalvs) [inline admin/partials/admin-monk-term-translation.php]: Localize label `esc_html_e( 'Add a translation +' )`

**Review by brenoalvs — CHANGES_REQUESTED** [review]

File: admin/partials/admin-monk-term-translation.php (hunk 3 above)
Comment (leonofre) [reply admin/partials/admin-monk-term-translation.php:48]: This is related to issue #79

**Review by brenoalvs — APPROVED** [review]

**Pull request merged**